emukit/bayesian_optimization/acquisitions/expected_improvement.py

```diff
@@ -88,3 +90,338 @@ def get_standard_normal_pdf_cdf(x: np.array, mean: np.array, standard_deviation:
     pdf = scipy.stats.norm.pdf(u)
     cdf = scipy.stats.norm.cdf(u)
     return u, pdf, cdf
+
+
+class QExpectedImprovement(ExpectedImprovement):
```

Comment:
Is that an accepted name in the literature? I mean, it isn't super descriptive, but it's ok to keep if that's what people use

Reply:
The paper refers to it at as "multipoint Expected Improvement", then later in the text they shorten in to q-EI for brevity. For us (as we don't need brevity) "MultipointExpectedImprovement" might be the better choice.

Reply:
thanks for chiming in, @mmahsereci , MultipointExpectedImprovement is certainly more expressive

Reply:
Fixed now